chore: remove unused l1-contracts repo mirror from release flow (#24929)
AztecProtocol/aztec-packages: docs/docs-developers/docs/tutorials/js_tutorials/aave_bridge.md

This template is a community-maintained starter. If the repository is unavailable, you can set up a Hardhat project manually and add the `@aztec/*` Solidity remappings from the [cross-chain messaging docs](../../foundational-topics/ethereum-aztec-messaging/index.md).

> If the template still pins `@aztec/l1-contracts` as a git dependency, replace it in `package.json` with the `@aztec/l1-artifacts` npm package at the version matching your Aztec version, e.g.:

Author: ```suggestion
You may need to replace the `@aztec/l1-contracts` dependency in `package.json` with the `@aztec/l1-artifacts` npm package at the version matching your Aztec version, e.g.:
```
